bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/FlatRow.java

```diff
@@ -128,13 +128,22 @@ public boolean equals(Object obj) {
         return false;
       }
       Cell other = (Cell) obj;
-      return Objects.equal(family, other.family) &&
-          Objects.equal(qualifier, other.qualifier) &&
-          timestamp == other.timestamp &&
+      return hasEqualKeys(other) &&
```

Comment:
There are some typos in the class doc: "This class stores represents a single row. It's a flattened version of the data of a".